code/multisig/template/Account.sol

```diff
@@ -0,0 +1,168 @@
+        public
+        view
+        override
+        returns (bytes4 magic)
+    {
+        // TO BE IMPLEMENTED
+    }
+
+    function payForTransaction(
+        bytes32,
+        bytes32,
+        Transaction calldata _transaction
+    ) external payable override onlyBootloader {
+        // TO BE IMPLEMENTED
+    }
+
+    function prepareForPaymaster(
+        bytes32, // _txHash
+        bytes32, // _suggestedSignedHash
+        Transaction calldata _transaction
+    ) external payable override onlyBootloader {
+        // TO BE IMPLEMENTED
+    }
+
+    // This function verifies that the ECDSA signature is both in correct format and non-malleable
+    function checkValidECDSASignatureFormat(bytes memory _signature) internal pure returns (bool) {
+        if(_signature.length != 65) {
+            return false;
+        }
+
+        uint8 v;
+  bytes32 r;
+  bytes32 s;
+  // Signature loading code
+  // we jump 32 (0x20) as the first slot of bytes contains the length
+  // we jump 65 (0x41) per signature
+  // for v we load 32 bytes ending with v (the first 31 come from s) then apply a mask
+  assembly {
+   r := mload(add(_signature, 0x20))
+   s := mload(add(_signature, 0x40))
+   v := and(mload(add(_signature, 0x41)), 0xff)
+  }
+  if(v != 27 && v != 28) {
+            return false;
+        }
+
+  // EIP-2 still allows signature malleability for ecrecover(). Remove this possibility and make the signature
+        // unique. Appendix F in the Ethereum Yellow paper (https://ethereum.github.io/yellowpaper/paper.pdf), defines
+        // the valid range for s in (301): 0 < s < secp256k1n ÷ 2 + 1, and for v in (302): v ∈ {27, 28}. Most
+        // signatures from current libraries generate a unique signature with an s-value in the lower half order.
+        //
+        // If your library generates malleable signatures, such as s-values in the upper range, calculate a new s-value
+        // with 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141 - s1 and flip v from 27 to 28 or
+        // vice versa. If your library also generates signatures with 0/1 for v instead 27/28, add 27 to v to accept
+        // these malleable signatures as well.
+        if(uint256(s) > 0x7FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF5D576E7357A4501DDFE92F46681B20A0) {
+            return false;
+        }
+
+        return true;
+    }
+
+    function extractECDSASignature(bytes memory _fullSignature) internal pure returns (bytes memory signature1, bytes memory signature2) {
+        require(_fullSignature.length == 130, "Invalid length");
+
+        signature1 = new bytes(65);
+        signature2 = new bytes(65);
+
+        // Copying the first signature. Note, that we need an offset of 0x20
+        // since it is where the length of the `_fullSignature` is stored
+        assembly {
+            let r := mload(add(_fullSignature, 0x20))
+   let s := mload(add(_fullSignature, 0x40))
+   let v := and(mload(add(_fullSignature, 0x41)), 0xff)
+
```

Comment:
Formatting seems off.

Reply:
it's really weird, just tabbing these lines over breaks the project & test with the estimateFee error. I tested this locally as well